projects/ad4052_ardz/coraz7s/system_bd.tcl

```diff
@@ -13,3 +13,5 @@ ad_ip_parameter rom_sys_0 CONFIG.PATH_TO_FILE "$mem_init_sys_file_path/mem_init_
 ad_ip_parameter rom_sys_0 CONFIG.ROM_ADDR_BITS 9
 
 sysid_gen_sys_init_file
+
+set_property strategy Performance_Retiming [get_runs impl_1]
```

Comment:
This usually goes in the system_project.tcl not _db (28 _project.tcl vs 2 _bd.tcl)